package/samplers/cmamae/sampler.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         self,
         *,
         param_names: list[str],
+        measure_names: list[str],
```

Comment:
I think line 39 would need to be updated since it mentions "Note that this sampler assumes the objective function will return a list of values". We would also need to add `measure_names` to the args above.